app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -647,7 +658,7 @@ def vm_compute_attributes(vm)
         interface_attrs = {}
         interface_attrs[:compute_attributes] = {}
         interface_attrs[:mac] = interface.mac
-        interface_attrs[:compute_attributes][:network] = network.name
+        interface_attrs[:compute_attributes][:network] = network.id
```

Comment:
This will cause the API to return network id in format `dvportgroup-123` or similar, instead of human readable network name.